charts/testkube/values.yaml

```diff
@@ -355,6 +355,24 @@ testkube-api:
     digest: ""
     # -- Testkube API k8s secret for private registries
     pullSecrets: []
+
+  # Test Workflows image parameters
+  imageWorkflows:
+    # -- Test Workflows image registry
+    registry: docker.io
```

Comment:
Will it work for default registry (docker) if it is undefined ("")? There Is some logic in Workflows for private registry access and I think the default Dockerhub registry URL is different

Reply:
this is how it is defined for api-server and it works, @rangoo94 any concerns about this?

Reply:
I think that it will work with `docker.io` default, same as for others 👍